test/request-response-samples/UpdateProfile__correct_namespaces_for_elements_with_base_ignorednamespaces/request.xml

```diff
@@ -1 +1 @@
-<?xml version="1.0" encoding="utf-8"?><soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"  xmlns:tns="http://www.bigdatacollect.or" xmlns:c="http://www.bigdatacollect.or/Common/Types" xmlns:n="http://www.bigdatacollect.or/Name/Types"><soap:Body><n:UpdateProfileRequest xmlns:n="http://www.bigdatacollect.or/Name/Types" xmlns="http://www.bigdatacollect.or/Name/Types"><n:Profile><n:IDs><c:UniqueID source="TESTSOURCE">100</c:UniqueID></n:IDs><n:Addresses><n:NameAddress><n:AddressLine xmlns:c="http://www.bigdatacollect.or/Common/Types">Another Address</n:AddressLine></n:NameAddress><n:NameAddress><n:AddressLine xmlns:c="http://www.bigdatacollect.or/Common/Types">My Address</n:AddressLine></n:NameAddress></n:Addresses><n:Phones><n:NamePhone primary="true"><n:PhoneData xmlns:c="http://www.bigdatacollect.or/Common/Types"><c:PhoneNumber>123</c:PhoneNumber></n:PhoneData></n:NamePhone><n:NamePhone primary="false"><n:PhoneData xmlns:c="http://www.bigdatacollect.or/Common/Types"><c:PhoneNumber>456</c:PhoneNumber></n:PhoneData></n:NamePhone></n:Phones></n:Profile></n:UpdateProfileRequest></soap:Body></soap:Envelope>
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?><soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"  xmlns:tns="http://www.bigdatacollect.or" xmlns:c="http://www.bigdatacollect.or/Common/Types" xmlns:n="http://www.bigdatacollect.or/Name/Types"><soap:Body><n:UpdateProfileRequest xmlns:n="http://www.bigdatacollect.or/Name/Types" xmlns="http://www.bigdatacollect.or/Name/Types"><Profile><IDs><UniqueID source="TESTSOURCE">100</UniqueID></IDs><Addresses><NameAddress><AddressLine>Another Address</AddressLine></NameAddress><NameAddress><AddressLine>My Address</AddressLine></NameAddress></Addresses><Phones><NamePhone primary="true"><PhoneData><PhoneNumber>123</PhoneNumber></PhoneData></NamePhone><NamePhone primary="false"><PhoneData><PhoneNumber>456</PhoneNumber></PhoneData></NamePhone></Phones></Profile></n:UpdateProfileRequest></soap:Body></soap:Envelope>
```

Comment:
Expected request in the test covering ignoreBaseNameSpaces option is wrong: while UpdateProfileRequest parent element correctly has namespace prefix along with its namespace definition, all its children have prefixes and namespace definitions as well. But the role of the ignoreBaseNameSpaces = true option is to suppress the prefixes in children.

The change in wsdl.js I am asking for is tested on my end in my interaction with an endpoint in Informatica's DataIntegration service. The endpoint is not able to validate the XML with these namespace related artifacts injected by the original wsdl.js. Meanwhile, SoapUI, as well as my version of wsdl.js, generate XML without the prefixes and able to connects to Informatica successfully.